src/views/splash/presentation.jsx

```diff
@@ -271,6 +275,17 @@ class SplashPresentation extends React.Component { // eslint-disable-line react/
             );
         }
 
+        if (this.props.sessionStatus === sessionActions.Status.FETCHED && Object.keys(this.props.user).length === 0) {
```

Comment:
We should really turn this into a `loggedIn` prop, or `isLoggedIn` method. Not in this PR, just a note — I see it's this way elsewhere...

Reply:
@rschamp, I think for ppl who don't have accs, should be able to see this:).